src/password/scripts/directives.js

```diff
@@ -3,9 +3,33 @@
 
     var module = angular.module('user_management.password');
 
+    module.service('catchErrors', [
```

Comment:
Can you update to use this service in all the other directives in this library? It's basically all of them except `sms-verification`.